srcpkgs/qtcreator/template

```diff
@@ -1,23 +1,24 @@
 # Template file for 'qtcreator'
 pkgname=qtcreator
-version=8.0.0
+version=9.0.1
 revision=1
 build_style=cmake
-configure_args="$(vopt_bool qbs BUILD_QBS)"
-hostmakedepends="clang llvm perl pkg-config python3 which qt5-qmake
- qt5-host-tools qt5-tools"
-makedepends="qt5-declarative-devel qt5-script-devel qt5-tools-devel
- qt5-quickcontrols qt5-svg-devel qt5-plugin-odbc qt5-plugin-sqlite
- qt5-plugin-tds qt5-plugin-pgsql qt5-plugin-mysql
+configure_args="$(vopt_bool qbs BUILD_QBS) -DWITH_DOCS=ON"
+hostmakedepends="clang llvm perl pkg-config python3 which
+ qt6-base-devel qt6-tools qtchooser"
```

Comment:
```suggestion
 qt6-base-devel qt6-tools-devel qtchooser"
```
`qt6-tools-devel` can also be removed from makedepends.
Furthermore add `qt6-shadertools-devel qt6-declarative-devel` has to be added to hostmakedepends.

Reply:
Scratch that, qt6-tools-devel is also linked against, so the trick to just only have it in host doesn't help.